app/models/domain.rb

```diff
@@ -51,7 +51,7 @@ class Domain < ApplicationRecord
 
   set_crud_hooks :domain
 
-  apipie :class, desc: "A class representing #{model_name.human} object" do
+  apipie :class, desc: "A class representing #{humanize_class_name} object" do
```

Comment:
I just realized this actually doesn't exist and we need something else to describe the object.

Reply:
Well, since the class names mostly taken as is for DSL, unless they explicitly renamed, it's okay to leave this untranslated and closer to something that is used within methods/return values. Although, there is a need to review all DSL documentation and make sure it's actually consistent.

Also, this whole string is not being translated...